diff --git a/src/main/java/io/simplelocalize/cli/configuration/ConfigurationLoader.java b/src/main/java/io/simplelocalize/cli/configuration/ConfigurationLoader.java
index 16fcd06..b741b5b 100644
--- a/src/main/java/io/simplelocalize/cli/configuration/ConfigurationLoader.java
+++ b/src/main/java/io/simplelocalize/cli/configuration/ConfigurationLoader.java
@@ -3,6 +3,7 @@ package io.simplelocalize.cli.configuration;
 import io.simplelocalize.cli.exception.ConfigurationException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.LoaderOptions;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 
@@ -36,7 +37,8 @@ public final class ConfigurationLoader
   private Configuration load(Path configurationFilePath)
   {
     File file = new File(URLDecoder.decode(String.valueOf(configurationFilePath.toFile()), StandardCharsets.UTF_8));
-    Constructor yamlTargetClass = new Constructor(Configuration.class);
+    LoaderOptions loaderOptions = new LoaderOptions();
+    Constructor yamlTargetClass = new Constructor(Configuration.class, loaderOptions);
     Yaml yaml = new Yaml(yamlTargetClass);
 
     log.info("Loading configuration file from: {}", configurationFilePath);
