diff --git a/src/main/java/xdev/tableexport/export/ReportBuilder.java b/src/main/java/xdev/tableexport/export/ReportBuilder.java
index 4e91b33..c57c796 100644
--- a/src/main/java/xdev/tableexport/export/ReportBuilder.java
+++ b/src/main/java/xdev/tableexport/export/ReportBuilder.java
@@ -366,7 +366,7 @@ public class ReportBuilder
 			return;
 		}
 		
-		textField.getLineBox().getPen().setLineWidth(border.getLineWidth());
+		textField.getLineBox().getPen().setLineWidth(Float.valueOf(border.getLineWidth()));
 		textField.getLineBox().getPen().setLineColor(border.getLineColor());
 		textField.getLineBox().getPen().setLineStyle(border.getLineStyle().getLineStyleEnum());
 	}
