diff --git a/src/test/java/org/camunda/bpm/extension/mockito/service/RuntimeServiceFluentMockTest.java b/src/test/java/org/camunda/bpm/extension/mockito/service/RuntimeServiceFluentMockTest.java
index 081e5d3..440427d 100644
--- a/src/test/java/org/camunda/bpm/extension/mockito/service/RuntimeServiceFluentMockTest.java
+++ b/src/test/java/org/camunda/bpm/extension/mockito/service/RuntimeServiceFluentMockTest.java
@@ -5,6 +5,7 @@ import org.camunda.bpm.engine.RuntimeService;
 import org.camunda.bpm.engine.task.Task;
 import org.camunda.bpm.engine.variable.Variables;
 import org.camunda.bpm.engine.variable.value.StringValue;
+import org.camunda.community.mockito.service.RuntimeServiceFluentMock;
 import org.camunda.community.mockito.task.TaskFake;
 import org.junit.Test;
 
