diff --git a/src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java b/src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java
index 5c571d0..05929d0 100755
--- a/src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java
+++ b/src/main/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManager.java
@@ -304,7 +304,6 @@ public class SauceConnectFourManager extends AbstractSauceTunnelManager implemen
      * @param destination the destination directory
      */
     private void extractArchive(AbstractUnArchiver unArchiver, File archive, File destination) {
-        unArchiver.enableLogging(new ConsoleLogger(Logger.LEVEL_DEBUG, "Sauce"));
         unArchiver.setSourceFile(archive);
         unArchiver.setDestDirectory(destination);
         unArchiver.extract();
